app/controllers/health_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HealthController < ApplicationController
+  def workers
+    status = worker_health_checker.all_healthy? ? :ok : :internal_error
+
+    render json: worker_health_checker.statuses, status: status
```

Comment:
Not sure what you are thinking in terms of URL structure, but I might do something like:

`health/workers` -> `Health::WorkersController#index`
`health/databases` -> `Health::DatabasesController#index`

Since there is a collection of stats at this endpoint, I'd expect it to be an `index` action. Future endpoints could also be `index` and namespaced within `Health` so they are grouped logically